package.json

```diff
@@ -88,7 +91,11 @@
     "styled-system": "5.1.5",
     "uuid": "^8.2.0"
   },
+  "resolutions": {
+    "@types/react": "^16.9.35"
```

Comment:
The `@types/react` should be obsolete here as react package have it own types definitions.